gencred/README.md

```diff
@@ -7,7 +7,148 @@
 
 ## Usage
 
-### Generate a service account token for a cluster. 
+### Script
+
+Run using Bazel:
+
+```shell
+$ bazel run //gencred/cmd -- <options>
+```
+
+Run using Golang:
+
+```console
+$ go run ./gencred/cmd <options>
```

Comment:
`k8s.io/test-infra` will be more portable than `.`

I'm still not a fan of the cmd suffix, everything in this repo uses either nothing (kubetest) or else a cmd prefix (phaino)

Why does this need to avoid the prow directory? When we move prow into its own repo, won't we want this utility to come along with it?

Reply:
I will drop the `cmd` suffix. 

I am avoiding `/prow` directory mainly because this is not coupled to Prow. A `kubeconfig` is a general, standardized format. If it was dependent on Prow libraries (or vice versa) then maybe it would make sense IMO.